extensions/empty-screenshots/src/index.ts

```diff
@@ -2,22 +2,42 @@ import { readdir, stat, rm } from "fs/promises";
 import { join, resolve } from "path";
 import { homedir } from "os";
 import { showToast, Toast, showHUD, getPreferenceValues } from "@raycast/api";
+import { exec } from "child_process";
 
 interface Preferences {
+  screenshotBehavior: string;
   daysToKeep: string;
   screenshotFolder: string;
+  prefixString: string;
+}
+
+async function moveToTrash(filePath: string) {
+  return new Promise((resolve, reject) => {
+    exec(`osascript -e 'tell application "Finder" to delete POSIX file "${filePath}"'`, (error, stdout, stderr) => {
+      if (error) {
+        console.error("Error moving file to Trash:", stderr);
+        reject(error);
+      } else {
+        console.log("File moved to Trash:", stdout);
+        resolve(stdout);
+      }
+    });
+  });
 }
 
 export default async function main() {
-  const { daysToKeep, screenshotFolder } = getPreferenceValues<Preferences>();
+  const { daysToKeep, screenshotFolder, screenshotBehavior, prefixString } = getPreferenceValues<Preferences>();
   const numberOfDays = parseInt(daysToKeep, 10);
-  const folder = resolve(screenshotFolder.replace("~", homedir));
+  const folder = resolve(screenshotFolder.replace("~", homedir()));
 
   try {
     const files = await readdir(folder);
 
     const screenshots = files
       .filter((file) => file.endsWith(".png") || file.endsWith(".mov"))
+      .filter((file) =>
+        prefixString ? file.startsWith(prefixString) : file.startsWith("CleanShot") || file.startsWith("Screen"),
```

Comment:
I just tried to change language to danish and the name here is `Skærmbillede 2024-05-22 kl. 09.08.22.png` - this will never be removed, so maybe it's better to do as before and only filter if custom `prefixString` has been set (which we could make to a comma seperated list)

Reply:
Are you still around @robotdestroy

Reply:
Yup, will clean this up this week!